### ISSUE TITLE
Denoise (profiled) : Noise profiles not loading for Nikon DSLR cameras (JPG pictures)

### ISSUE THREAD

**Issue by GLLM** [darktable-org/darktable#6644]
Hello all,

I'm running Dt 3.2.1 on Linux  Kubuntu 20.10

I'm using **_Denoise (Profiled)_** module and it works perfectly with my Fujifilm X-T3 camera.

But, when I'm retouching photos (JPG) taken with (not owned anymore) Nikon DSLRs, the profile is neither seen nor loaded/loadable in this module.

![image](https://user-images.githubusercontent.com/3989573/97211701-b652bc80-17bf-11eb-854d-97c51a822ec1.png)

I've looked into _noiseprofiles.json file_ and wonder if this could be caused by the fact that :
* Nikon maker is spelled "Nikon", whereas any of my pictures taken with Nikon DSLR have their maker set (never manually changed) as "NIKON CORPORATION"
* Nikon DSLR are named as Dxxxx where any of my pictures are showing "NIKON Dxxxx"

Camera & Maker as visible on one of my pic :
![image](https://user-images.githubusercontent.com/3989573/97211174-ed749e00-17be-11eb-84f4-aa4f0d3b159f.png)

Camera list of my collection
![image](https://user-images.githubusercontent.com/3989573/97211084-cf0ea280-17be-11eb-88e3-29b7fb3d5ea5.png)

I've therefore edited _/usr/share/darktable/noiseprofiles.json_ and  I've tested changing "Nikon" maker to "NIKON CORPORATION" and "D600" to "NIKON D600" and ....
hurrah, the camera is recognized and therefore the noise profile is visible and can be loaded.

![image](https://user-images.githubusercontent.com/3989573/97211268-14cb6b00-17bf-11eb-8167-01e5d9bb58f7.png)
![image](https://user-images.githubusercontent.com/3989573/97211301-20b72d00-17bf-11eb-82cd-984e683b6ef6.png)

**Questions**
* How do you see both Nikon maker shown and the camera named in the image information module (left pane) ?
* Anyone is not able to use noise profiles with JPG pictures taken with Nikon DSLRs ?
* Can you fix this

Usual addendum : 
* running Kubuntu 20.10, but issue was the same on K20.04, K19.10, K19.04 ...
* I have wasy enough RAM (32GB) and a healthy CPU i7700K + run with OpenCL on GFX 1070.

Thanks
GLLM

**Comment by GLLM** [comment]
Up !

**Comment by johnny-bit** [comment]
I think this might be unfixable by darktable but rather it might be a combination of couple issues:

- I wouldn't do denoising on JPEGs as camera usually denoises those + jpeg compression also does some denoising
- the exif data seems to be different enough on RAWs and JPEGs for exiv2 to misrepresent/mistranslate those files
- maybe there was some change along the way with jpegs because I checked some Nikon jpegs from my friends and haven't got a problem with them... so I essentially cannot replicate.

How about this - try to remove and then re-import images, making sure you have recent enough verson on exiv2. If that helps - please close the issue :)

**Comment by GLLM** [comment]
Hi @johnny-bit !
Thanks for your reply !

It happens that I had to rename (externally) 10k+ pictures taken with my previous Nikon DSLR ... hence I had to re-import them in dt. And unfortunately, this did not solve anything !

About JPEG : the denoise (profile), when used wisely & lightly, can help greatly ... even on JPEGs :-)

I just reinstalled exiv2 0.27.3 ... to no avail.

Indeed, on a RAW (NEF) file, it works ok.
In fact, the Maker & Model (and evn lens) info are different between the NEF and the JPEG files ! Not sure why, as I did not change any of them : 
 
JPEG
![image](https://user-images.githubusercontent.com/3989573/106563966-e3d0a400-652c-11eb-94ba-bec768b9e53b.png)

NEF (RAW)
![image](https://user-images.githubusercontent.com/3989573/106564085-082c8080-652d-11eb-8905-d9eb33d48ee8.png)

So, I'm indeed not sure how to deal with this.

Oh, and I don't know why, but Lens correction module does not bother : it works with both

Thanks
GLLM
